.travis.yml

```diff
@@ -27,7 +27,7 @@ matrix:
       scala: 2.11.7
       env: TEST_SPARK_VERSION="1.5.0"
 script:
-  - sbt -Dspark.testVersion=$TEST_SPARK_VERSION ++$TRAVIS_SCALA_VERSION coverage test
```

Comment:
I just wanted to set up this later and properly for this issue https://github.com/databricks/spark-xml/issues/1.